x-pack/plugins/observability_solution/apm/common/es_fields/es_fields.test.ts

```diff
@@ -20,7 +20,7 @@ const fieldnames = {
 
 describe('Transaction', () => {
   const transaction: AllowUnknownProperties<Transaction> = {
-    '@timestamp': new Date().toString(),
+    '@timestamp': new Date('01/01/2024').toISOString(),
```

Comment:
you'll run into timezone issues with this on CI. you'll need to use an ISO timestamp with a timezone.

Reply:
I tried different formats but you are right about the timezone here, to simplify this I converted the `'@timestamp'`  to string to make the test more stable.